src/borg/archiver.py

```diff
@@ -2254,6 +2254,15 @@ def process_epilog(epilog):
         '\*/.bundler/gems' to get the same effect. See ``borg help patterns`` for
         more information.
 
+        In addition to using ``--exclude`` patterns, it is possible to use
+        ``--exclude-if-present`` to specify the name of a filesystem object (e.g. a file
+        or folder name) which, when contained within another folder, will prevent the
+        containing folder from being backed up.  By default, the containing folder and
+        all of its contents will be omitted from the backup.  If, however, you wish to
+        only include the objects specified by ``--exclude-if-present`` in your backup,
+        and not include any other contents of the containing folder, this can be enabled
+        through using the ``--keep-exclude-tags`` option.
```

Comment:
well, this is explaining what it does, but not much the motivation to do so.
- for tag files, that is just not to lose the tags (the information about not wanting to back up this specific dir) in case you have to recover from a backup.
- for tag directories with actual content, there is the .git/ usecase: you do not want to back up all your workdir checkouts, but you do want to backup your repos.